tools/ldcontext_generator.py

```diff
@@ -149,22 +177,22 @@ def schema_2_ld_context(schema, uri_prefix, predefined_mappings):
         all_properties, uri_prefix, predefined_mappings)
 
     if entity_type is not None:
-        ld_context[entity_type] = uri_prefix + '/' + 'entity-types' + '/' + entity_type
+        ld_context[entity_type] = uri_prefix + '#' + entity_type
 
     return ld_context
 
 
-def process_file(input_file, uri_prefix, predefined_mappings):
+def process_file(input_file, uri_prefix, predefined_mappings, terms_mappings):
     if os.path.isfile(input_file) and input_file.endswith('schema.json'):
         print(input_file)
-        aggregate_ld_context(input_file, uri_prefix, predefined_mappings)
+        aggregate_ld_context(input_file, uri_prefix, predefined_mappings, terms_mappings)
     elif os.path.isdir(input_file):
         for f in (os.listdir(input_file)):
             process_file(os.path.join(input_file, f),
-                         uri_prefix, predefined_mappings)
+                         uri_prefix, predefined_mappings, terms_mappings)
 
 
-def aggregate_ld_context(f, uri_prefix, predefined_mappings):
+def aggregate_ld_context(f, uri_prefix, predefined_mappings, terms_mappings):
     global aggregated_context
```

Comment:
we need to also declare as global terms_list, don't we?

Reply:
Fixed in 8933b1ef27836a637fd56c3530e5b226b30a2fdb